tests/tests/payloads/issue_comment___pr.json

```diff
@@ -0,0 +1,225 @@
+    "created_at": "2020-10-21T09:18:09Z",
+    "updated_at": "2021-06-09T14:15:26Z",
+    "pushed_at": "2021-06-16T15:23:59Z",
+    "git_url": "git://github.com/mendersoftware/deviceconnect.git",
+    "ssh_url": "git@github.com:mendersoftware/deviceconnect.git",
+    "clone_url": "https://github.com/mendersoftware/deviceconnect.git",
+    "svn_url": "https://github.com/mendersoftware/deviceconnect",
+    "homepage": "https://mender.io",
+    "size": 5432,
+    "stargazers_count": 2,
+    "watchers_count": 2,
+    "language": "Go",
+    "has_issues": false,
+    "has_projects": false,
+    "has_downloads": true,
+    "has_wiki": false,
+    "has_pages": false,
+    "forks_count": 10,
+    "mirror_url": null,
+    "archived": false,
+    "disabled": false,
+    "open_issues_count": 1,
+    "license": {
+      "key": "other",
+      "name": "Other",
+      "spdx_id": "NOASSERTION",
+      "url": null,
+      "node_id": "MDc6TGljZW5zZTA="
+    },
+    "forks": 10,
+    "open_issues": 1,
+    "watchers": 2,
+    "default_branch": "master"
+  },
+  "organization": {
+    "login": "mendersoftware",
+    "id": 15040539,
+    "node_id": "MDEyOk9yZ2FuaXphdGlvbjE1MDQwNTM5",
+    "url": "https://api.github.com/orgs/mendersoftware",
+    "repos_url": "https://api.github.com/orgs/mendersoftware/repos",
+    "events_url": "https://api.github.com/orgs/mendersoftware/events",
+    "hooks_url": "https://api.github.com/orgs/mendersoftware/hooks",
+    "issues_url": "https://api.github.com/orgs/mendersoftware/issues",
+    "members_url": "https://api.github.com/orgs/mendersoftware/members{/member}",
+    "public_members_url": "https://api.github.com/orgs/mendersoftware/public_members{/member}",
+    "avatar_url": "https://avatars.githubusercontent.com/u/15040539?v=4",
+    "description": "Mender is an end-to-end open source update manager for IoT"
+  },
+  "sender": {
+    "login": "alfrunes",
+    "id": 35835176,
+    "node_id": "MDQ6VXNlcjM1ODM1MTc2",
+    "avatar_url": "https://avatars.githubusercontent.com/u/35835176?v=4",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/alfrunes",
+    "html_url": "https://github.com/alfrunes",
+    "followers_url": "https://api.github.com/users/alfrunes/followers",
+    "following_url": "https://api.github.com/users/alfrunes/following{/other_user}",
+    "gists_url": "https://api.github.com/users/alfrunes/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/alfrunes/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/alfrunes/subscriptions",
+    "organizations_url": "https://api.github.com/users/alfrunes/orgs",
+    "repos_url": "https://api.github.com/users/alfrunes/repos",
+    "events_url": "https://api.github.com/users/alfrunes/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/alfrunes/received_events",
+    "type": "User",
+    "site_admin": false
+  }
+}
```

Comment:
Nit:
Missing newline

Reply:
added (there was no new line in the other json)